grunt/tasks/schema-defaults.js

```diff
@@ -57,7 +57,7 @@ module.exports = function(grunt) {
         //iterate through lanugage folders
         grunt.file.expand({filter: 'isDirectory'}, grunt.config('outputdir') + 'course/*').forEach(function(path) {
             var currentCourseFolder = path;
-            var currentCourseJsonFile = currentCourseFolder + '/' + 'course.json';
+            var currentCourseJsonFile = currentCourseFolder + '/' + 'course.' + grunt.config('jsonext');
```

Comment:
Should we be caching `jsonext` outside the `forEach` loop like [here](https://github.com/adaptlearning/adapt_framework/pull/1684/files#diff-3bac7cf018801ee7c4228dce79ba0de7R12)?